internals-js/src/federation.ts

```diff
@@ -1255,25 +1249,13 @@ export class FederationMetadata {
     return this.getPost20FederationDirective(FederationDirectiveName.POLICY);
   }
 
-  sourceAPIDirective(): Post20FederationDirectiveDefinition<SourceAPIDirectiveArgs> {
-    return this.getPost20FederationDirective(FederationDirectiveName.SOURCE_API);
-  }
-
-  sourceTypeDirective(): Post20FederationDirectiveDefinition<SourceTypeDirectiveArgs> {
-    return this.getPost20FederationDirective(FederationDirectiveName.SOURCE_TYPE);
-  }
-
-  sourceFieldDirective(): Post20FederationDirectiveDefinition<SourceFieldDirectiveArgs> {
-    return this.getPost20FederationDirective(FederationDirectiveName.SOURCE_FIELD);
-  }
-
-  fromContextDirective(): Post20FederationDirectiveDefinition<{ field: string }> {
+fromContextDirective(): Post20FederationDirectiveDefinition<{ field: string }> {
```

Comment:
nit: this indentation seems wrong?

Reply:
Yeah... this project isn't set up to format everything with Prettier. I thought I fixed that in #3050 but that only reformats tests? I don't know enough about Prettier to try to fix this for real so, unfortunately, I gotta do more manual formatting 💀